api/v1alpha1/warehouse_types.go

```diff
@@ -363,8 +365,15 @@ type ImageSubscription struct {
 	// changes.
 	// More info: https://github.com/masterminds/semver#checking-version-constraints
 	//
+	// Deprecated: Use constraint instead.
```

Comment:
In other spots where we've done this, I notice we've capitalized the field name to match Go syntax, rather than how it presents in YAML.

```suggestion
	// Deprecated: Use Constraint instead.
```